doc/crypt_gensalt.3

```diff
@@ -250,14 +266,18 @@ l l l.
 Interface	Attribute	Value
 T{
 .Nm crypt_gensalt
-T}	Thread safety	MT-Unsafe race:crypt_gensalt
+T}	Thread safety	MT-Safe*
 T{
 .Nm crypt_gensalt_rn ,
 .Nm crypt_gensalt_ra
 T}	Thread safety	MT-Safe
 .TE
 .ad
-.sp
+.Pp
+\&* In this implementation.
+Do NOT rely on this to be portable behaviour.
```

Comment:
libxcrypt currently mostly uses the `behavior` spelling (not `behaviour`), so maybe it's best to use that consistently.

Reply:
Fixed.